docs/running-on-kubernetes.md

```diff
@@ -561,6 +561,13 @@ specific to Spark on Kubernetes.
     This is distinct from <code>spark.executor.cores</code>: it is only used and takes precedence over <code>spark.executor.cores</code> for specifying the executor pod cpu request if set. Task 
     parallelism, e.g., number of tasks an executor can run concurrently is not affected by this.
 </tr>
+<tr>
+  <td><code>spark.kubernetes.executor.maxInitFailures</code></td>
+  <td>10</td>
```

Comment:
instead of a fixed number, should this be expressed as a factor on the total number of executors?